internal/controller/pkg/runtime/runtime_function.go

```diff
@@ -182,14 +182,36 @@ func (h *FunctionHooks) Deactivate(ctx context.Context, _ v1.PackageRevisionWith
 	return nil
 }
 
-func functionDeploymentOverrides(image string) []DeploymentOverride {
+func functionDeploymentOverrides(pr v1.PackageRevisionWithRuntime, image string) []DeploymentOverride {
 	do := []DeploymentOverride{
 		DeploymentRuntimeWithAdditionalPorts([]corev1.ContainerPort{
 			{
 				Name:          GRPCPortName,
 				ContainerPort: GRPCPort,
 			},
 		}),
+		DeploymentRuntimeWithAdditionalEnvironments([]corev1.EnvVar{
```

Comment:
Is there no spec for the runtime that also needs to be updated? This will only apply to deployments.

Reply:
We only need this in deployments, so that provider/function processes can access these values if they need to.